Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsActivity.java

```diff
@@ -211,11 +218,19 @@ public void showLoading()
   {
     if (dialog == null)
     {
-      dialog = new ProgressDialog(this);
-      dialog.setMessage(getString(R.string.load_settings));
-      dialog.setIndeterminate(true);
+      ProgressBar progressBar =
+              new ProgressBar(this, null, android.R.attr.progressBarStyle);
+      final int dpHorizontal = 24;
+      final int dpVertical = 16;
+      int pxHorizontal = ThemeHelper.dpToPx(dpHorizontal, getResources());
+      int pxVertical = ThemeHelper.dpToPx(dpVertical, getResources());
+      progressBar.setPadding(pxHorizontal, pxVertical, pxHorizontal, pxVertical);
```

Comment:
Could it be worth defining a layout XML file for this instead of doing it all in code?

Reply:
Good point, that'll be in the next few commits